dotnet/targets/Xamarin.Shared.Sdk.props

```diff
@@ -91,6 +91,8 @@
 		<!-- Tell .NET to skip sanity checks that trigger too eagerly for builds with multiple RuntimeIdentifiers -->
 		<AllowPublishAotWithoutRuntimeIdentifier Condition="'$(AllowPublishAotWithoutRuntimeIdentifier)' == '' And '$(_UseNativeAot)' == 'true'">true</AllowPublishAotWithoutRuntimeIdentifier>
 		<AllowSelfContainedWithoutRuntimeIdentifier Condition="'$(AllowSelfContainedWithoutRuntimeIdentifier)' == ''">true</AllowSelfContainedWithoutRuntimeIdentifier>
+
+		<XcsyncPath Condition="'$(XcsyncPath)' == ''">$(_XamarinSdkRootDirectoryOnMac)tools/lib/xcsync/xcsync.dll</XcsyncPath>
```

Comment:
Originally it was meant for that. However, the script redirects to an architecture specific executable, which is problematic since you cannot create a universal launcher for NET8.0, or at least I was unable to make that work, which means we would have to have two launchers, one x64 and arm64. since the tool isn't on the path to begin with, and we had to use a target to run it, using dotnet to run the assembly was the better option. This can be cleaned up some.

Reply:
The script could just do "dotnet path/to/xcsync.dll"